build/common.props

```diff
@@ -5,7 +5,7 @@
     <LangVersion>latest</LangVersion>
     <MajorVersion>4</MajorVersion>
     <MinorVersion>15</MinorVersion>
-    <PatchVersion>1</PatchVersion>
+    <PatchVersion>2</PatchVersion>
```

Comment:
We had a windows-only hotfix as `4.15.2`. Should this be `4.15.3`?

https://github.com/Azure/azure-functions-host/releases/tag/v4.15.2

Reply:
Ah, missed that! Yes, having two with the same version would create some confusion. Let me update.

Reply:
Done.

Reply:
change PR title /commit message to reflect new version when merging to avoid confusion